docs/integrations/fastapi.md

```diff
@@ -102,6 +102,40 @@ To avoid tracing certain URLs, you can specify a string of comma-separated regex
 - The environment variable `OTEL_PYTHON_FASTAPI_EXCLUDED_URLS`.
 - The environment variable `OTEL_PYTHON_EXCLUDED_URLS` (which will also apply to other instrumentation).
 
+!!! example
+    If you'd like to trace all URLs except the base `/` URL, you can use the following regex: `^https?://[^/]+/$`
```

Comment:
Are all URLs guaranteed to end with a `/`? Like, I'd think you'd want `^https?://[^/]+` as your regex (or `^https?://[^/]+.*$` if it needs the trailing `$`)

Reply:
Yes good call, I've changed to `^https?://[^/]+`